pyproject.toml

```diff
@@ -55,15 +54,30 @@ docs = [
     "numpydoc",
     "tomli; python_version < '3.11'"
 ]
-
 # we can use self-references to simplify all
 all = [
     "nectarchain[test,docs,dev]",
 ]
 
+
+[tool.pytest.ini_options]
+addopts = "--ignore=src/nectarchain/user_scripts"
+
+
+
 [tool.setuptools.packages.find]
 where = ["src"]
 exclude = ["nectarchain._dev_version"]
 
 [tool.setuptools_scm]
 write_to = "src/nectarchain/_version.py"
+
+[tool.ruff.lint]
```

Comment:
I don't really get why `roof` options are added here. In CI, we still use flake8 and not yet ruff.

Reply:
Same as for your comment about black, I use ruff to format module. Ruff and Black are complementary